android/src/main/java/vn/luongvo/kmm/survey/android/ui/screens/login/LoginScreen.kt

```diff
@@ -32,25 +36,38 @@ private const val BlurRadius = 25f
 private val LogoOffset = Offset(0f, -229f)
 
 @Composable
-fun LoginScreen(viewModel: LoginViewModel = getViewModel()) {
+fun LoginScreen(
+    viewModel: LoginViewModel = getViewModel(),
+    navigator: (destination: AppDestination) -> Unit
+) {
     var email by remember { mutableStateOf("") }
     var password by remember { mutableStateOf("") }
+    val isLoading by viewModel.isLoading.collectAsState()
+    val isLoggedIn by viewModel.isLoggedIn.collectAsState()
 
-    LoginScreenContent(
-        email = email,
-        password = password,
-        onEmailChange = {
-            email = it
-            // TODO https://github.com/luongvo/kmm-survey/issues/8
-        },
-        onPasswordChange = {
-            password = it
-            // TODO https://github.com/luongvo/kmm-survey/issues/8
-        },
-        onLogInClick = {
-            // TODO https://github.com/luongvo/kmm-survey/issues/8
-        },
-    )
+    val context = LocalContext.current
+    LaunchedEffect(viewModel.error) {
+        viewModel.error.collect { error -> error.showToast(context) }
```

Comment:
[Optional] I prefer to use a `Composable` alert in this case. You will see some differences between using Composable alert with toast 😆

Reply:
@suho Possible, but I would like to simply showing a Toast, instead of an AlertDialog, can't I? 😂

Reply:
Depend on you 🤡. But normally, on client projects, we will show an alert. You can try it later on the home screen 😆

Reply:
Sure, thanks for the suggestion @suho, I will consider using it in the corresponding context.

Reply:
@suho I think I will migrate to AlertDialog lol, can not test the Toast in part 2 right now 😮‍💨 https://github.com/android/android-test/issues/803